Makefile

```diff
@@ -1,7 +1,7 @@
 all:    |   clean-dependencies  test
 
 deps:	clean
-	npm	install
+	npm	install --no-optional
```

Comment:
This is here because of `ganache-cli`, which has `scrypt` as an `optionalDependency` and breaks `npm install`